certified-connectors/SkyPointCloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,840 @@
+                          "description": "name"
+                        },
+                        "refreshtime": {
+                          "type": "string",
+                          "description": "refreshtime"
+                        }
+                      }
+                    },
+                    "description": "partitions"
+                  },
+                  "schemaDescription": {
+                    "type": "string",
+                    "description": "schemaDescription"
+                  },
+                  "modifiedTime": {
+                    "type": "string",
+                    "description": "modifiedTime"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Entities",
+        "description": "This operation gets entities from an instance",
+        "operationId": "GetEntities",
+        "parameters": [
+          {
+            "name": "tenantId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetTenants",
+              "value-path": "id",
+              "value-title": "businessName",
+              "parameters": {
+                "Ocp-Apim-Subscription-Key": "7e83fc21dcb44d1db06b8284b0c0fb89"
+              }
+            },
+            "description": "Identifier of a tenant",
+            "x-ms-summary": "Tenant Id"
+          },
+          {
+            "name": "instanceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Identifier of an instance",
+            "x-ms-summary": "Instance Id",
+            "x-ms-dynamic-values": {
+              "operationId": "GetInstances",
+              "value-path": "id",
+              "value-title": "name",
+              "parameters": {
+                "tenantId": {
+                  "parameter": "tenantId"
+                },
+                "Ocp-Apim-Subscription-Key": "7e83fc21dcb44d1db06b8284b0c0fb89"
+              }
+            }
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "default": "id,name"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "7e83fc21dcb44d1db06b8284b0c0fb89",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/instances/": {},
+    "/instances/{instanceId}/data/{entityName}": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@vaibhavrmore1 Please replace ALL the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Refer - [Documentation](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-responses)